lab-andrew/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This is good, there were a few issues with your devDependencies.  Please see my comments below.